.github/workflows/build-mobile.yml

```diff
@@ -111,14 +113,14 @@ jobs:
           TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
           TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
         run: pnpm build:llm
-      # - uses: ledgerhq/actions/get-package-infos@main
-      #   id: post-version
-      #   with:
-      #     path: ${{ github.workspace }}/apps/ledger-live-mobile
+      - uses: ledgerhq/ledger-live/tools/actions/get-package-infos@develop
+        id: post-version
+        with:
+          path: ${{ github.workspace }}/apps/ledger-live-mobile
       - name: upload Android app
         uses: actions/upload-artifact@v3
         with:
-          name: ${{ steps.version.outputs.version }}-release
+          name: ${{ steps.post-version.outputs.version }}-release.apk
```

Comment:
remove the `.apk` because since we are uploading a folder it will have .zip by default

Reply:
Done f809a45363df3c33b900219219c50929f85e6e5e